### ISSUE TITLE
`supabase start` doesn't pull images if they don't exist

### ISSUE THREAD

**Issue by DanielGiljam** [supabase/cli#493]
# Bug report

## Describe the bug

After upgrading the CLI and running `supabase start`, the command fails with an error saying "no such image".

This is the exact error message:

```
Error restoring branch main: Branch was not dumped.
Failed to restart storage-api: Error response from daemon: No such container: supabase_storage_example-project
Error: Error response from daemon: No such image: public.ecr.aws/t3w2s2c9/storage-api:v0.20.1
```

I tried with a clean project as well (delete `supabase/` directory, run `supabase init`). Same error.

I started manually pulling the images one by one, re-running `supabase-start`, getting a new error message for which image next to pull. So the first image it errored with wasn't actually `storage-api`. The first image it errored with was `gotrue`, if I remember correctly.

## To Reproduce

1. Upgrade CLI to v1.5.2 (`brew upgrade supabase`)
2. Run `supabase stop` (if the development environment is still running)
3. Run `supabase start`
4. Realize there are some syntax errors in `seed.sql` which causes the command to fail. Fix those, then re-run `supabase start`
5. Witness "no such image" error

## Expected behavior

The CLI would automatically pull missing images, like I remember it doing initially when I started using it.

## System information

- OS: macOS

## Additional context

Could the bug have anything to do with any of the following things?

1. I had `supabase` running while upgrading?
2. When running `supabase start` after upgrading, there were syntax errors in `seed.sql` which caused the command to bail out early. Perhaps some state was stored somewhere and not invalidated which caused problems when re-running the command?

**Comment by sweatybridge** [comment]
It's a bad release on my part. Should be fixed in 1.5.3

**Comment by DanielGiljam** [comment]
Ok! Thanks for the quick response!
